packages/frontend/src/components/dataContracts/DataContractsListItem.js

```diff
@@ -0,0 +1,28 @@
+import {Link} from "react-router-dom";
+import './DataContractsListItem.scss'
+
+
+function DataContractsListItem ({dataContract, size = 'l'}) {
+    const identifier = dataContract.identifier
+    const timestamp = dataContract.timestamp
+    
+    return (
+        <>
+            {typeof identifier === 'string' &&
```

Comment:
I don't think you can render having an identifier, as it will always be loaded from the API